src/main/scripts/startMZmine_Windows.bat

```diff
@@ -62,5 +75,24 @@ set MAIN_CLASS=net.sf.mzmine.main.MZmineCore
 rem This command starts the Java Virtual Machine
 %JAVA_COMMAND% %JAVA_PARAMETERS% %MAIN_CLASS% %*
 
+
+rem Do not modify:
+rem Kill/cleanup remaining Rserve instances
+setlocal disableDelayedExpansion
+:: Load the file path "array"
+for /f "tokens=1* delims=:" %%A in ('dir /b %TMP_FILE_DIRECTORY%\rs_pid_*.pid^|findstr /n "^"') do (
+  set "file.%%A=%TMP_FILE_DIRECTORY%\%%B"
+  set "file.count=%%A"
+)
+:: Access the values
+setlocal enableDelayedExpansion
+for /l %%N in (1 1 %file.count%) do (
+  echo !file.%%N!
+  set /P pid=<!file.%%N!
+  echo !pid!
+  del !file.%%N!
+  taskkill /PID !pid! /F 2>nul
+)
+
 rem If there was an error, give the user chance to see it

```

Comment:
I think this "if ERRORLEVEL " code should go just after the %JAVA_COMMAND% line because we want to check the return code of the java instance, not that of the RServe killing.